src/machine/machine_rp2040_rom.go

```diff
@@ -1,4 +1,4 @@
-//go:build rp2040
+//go:build tinygo && rp2040
```

Comment:
What's special about this single machine .go file in PR?

Reply:
Good question. @aykevl could you please elaborate?

Reply:
@aykevl any comment on this before I merge it?

Reply:
It uses CGo, which leads to problems. None of the other files do.